work9/index.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+const ul = document.getElementById("js-list");
+const data = [{
+  to: "bookmark.html",
+  img: "1.png",
+  alt: "画像1",
+  text: "ブックマーク",
+},
+{
+  to: "message.html",
+  img: "2.png",
+  alt: "画像2",
+  text: "メッセージ"
+},
+];
+
+function getData() {
+  const result = new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve(data);
+
+    }, 3000);
+  })
+  return result;
+}
+
+async function promise() {
+  const imgArray = await getData();
+  createElement(imgArray);
+
+}
+
+function createElement(img) {
```

Comment:
IMO:
The argument name "img" might be confused ( with "img" in foreach ).

Reply:
Please confirm the variable name change.


970b70e